src/main/resources/typescript-axios-webpack-module-federation/api.mustache

```diff
@@ -1,8 +1,8 @@
 /* tslint:disable */
 /* eslint-disable */
 import type { AxiosStatic } from 'axios'
-import { APIFactory } from '@redhat-cloud-services/javascript-clients-shared/dist/utils';
-import { ApiConfig } from '@redhat-cloud-services/javascript-clients-shared/dist/common'
+import { APIFactory } from '@redhat-cloud-services/javascript-clients-shared/utils';
```

Comment:
@tahmidefaz I think if we make these changes in this PR - we also need to update the version of shared-utils in each package.json for each client: https://github.com/RedHatInsights/javascript-clients/blob/main/packages/patch/package.json#L26

However since this PR hasn't merged - version 2.0.0 won't be available in NPM yet 🤔

Reply:
@florkbr just removed the commit with the `/dist/` changes like we discussed. Let's see what happens.

Reply:
Looks like it just wanted the `package.lock` to be updated